controllers/verification.go

```diff
@@ -531,6 +531,35 @@ func (c *ApiController) VerifyCode() {
 		return
 	}
 
+	var application *object.Application
```

Comment:
we will use this api for other usage, check ip here is not a good idea, and this api can't help user to login

Reply:
you are right, i have changed the check here to set-password API. This is my mistake. I didn't correctly judge the core API of ForgetPage.